backend/wmg/pipeline/summary_cubes/marker_genes.py

```diff
@@ -57,10 +65,17 @@ def create_marker_genes_cube(corpus_path: str):
                 "tissue_ontology_term_ids": [tiss],
                 "organism_ontology_term_id": organism,
             }
-            t_markers = get_markers(target, context, corpus=corpus_path, test="ttest", percentile=0.05, n_markers=None)
-            b_markers = get_markers(
-                target, context, corpus=corpus_path, test="binomtest", percentile=0.3, n_markers=None
-            )
+            try:
+                t_markers = get_markers(
+                    target, context, corpus=corpus_path, test="ttest", percentile=0.05, n_markers=None
+                )
+                b_markers = get_markers(
+                    target, context, corpus=corpus_path, test="binomtest", percentile=0.3, n_markers=None
+                )
+            except MarkerGeneCalculationException as e:
```

Comment:
try/except so pipeline doesn't fail if no cells match query criteria

Reply:
this explanation would go well as a comment in the code